Project/Source/Panels/PanelScene.cpp

```diff
@@ -151,6 +151,7 @@ void PanelScene::Update() {
 		// Update viewport size
 		ImVec2 size = ImGui::GetContentRegionAvail();
 		if (App->renderer->GetViewportSize().x != size.x || App->renderer->GetViewportSize().y != size.y) {
+			// TODO, These should use the EVENT SCREEN_RESIZED
```

Comment:
I know this says //TODO, but does this mean that ScreenResized is being called both from here and from the TesseractEvent?

Reply:
Currently, the OnResize call from ModuleInput is only called when the Application Window is rescaled, and this one is called only when the viewport is resized. They are basically two different inputs that call the same function (ViewportResized of ModuleUserInterface)

Reply:
I see, lftm good job!